pkg/features/kube_features.go

```diff
@@ -304,6 +309,9 @@ var defaultVersionedFeatureGates = map[featuregate.Feature]featuregate.Versioned
 	MultiKueueAdaptersForCustomJobs: {
 		{Version: version.MustParse("0.14"), Default: false, PreRelease: featuregate.Alpha},
 	},
+	WorkloadRequestUseMergePatch: {
+		{Version: version.MustParse("0.14"), Default: true, PreRelease: featuregate.Alpha},
```

Comment:
```suggestion
		{Version: version.MustParse("0.14"), Default: false, PreRelease: featuregate.Alpha},
```
Don't forget to revert it.